add parameter to specify build configuration

lukaswuerzburger opened the pull request:
Solves #2

Commit:
add parameter to specify build configuration

lyndsey-ferguson commented:
This looks great! I should be able to run it locally later on today. Could you also update the test?
https://github.com/lyndsey-ferguson/get_product_bundle_id/blob/master/spec/get_product_bundle_id_action_spec.rb

Commit:
add tests for the build configuration parameter

- add build configurations 'Debug Stage' and 'Release Stage' in the demo app
- add target 'CoinTossingManager' in the demo app
- add scheme 'MoneyShakerManager'

lukaswuerzburger commented:
I've added 3 tests. Please check if they make sense.

Commit:
Fix some issues

Commit:
upgrade fixture project

lyndsey-ferguson commented:
I've got a fix waiting: https://github.com/truffls/get_product_bundle_id/pull/1

lukaswuerzburger committed:
Merge pull request #1 from lyndsey-ferguson/fix-add-build-config

Fix add build config
